test/simpleUpgrader.js

```diff
@@ -97,7 +97,7 @@ describe("SimpleUpgrader", function () {
       // create module V2
       const moduleV2 = await deployer.deploy(Module, {}, registry.contractAddress, false, 0);
       // register module V2
-      await registry.registerModule(moduleV2.contractAddress, formatBytes32String("V2"));
+      // await registry.registerModule(moduleV2.contractAddress, formatBytes32String("V2"));
```

Comment:
Did you intentionally leave this line commented out or did you want to delete it?

Reply:
This was intentionally done in [this commit](https://github.com/argentlabs/argent-contracts/pull/97/commits/92d2573ffafca0a1636a14201e0ac4233f44fbbd).